hometask6

```diff
@@ -0,0 +1,68 @@
+# Task 1
+#
+# import time
+# def my_decorator(func):
+#     def wrap():
```

Comment:
Use *args and **kwargs here and pass these to the `func` call to be able to decorate functions with arguments